pkg/config/env.go

```diff
@@ -11,6 +11,8 @@ var Config = struct {
 
 	// LogrusLevel sets the logrus logging level
 	LogrusLevel string `env:"FLAGR_LOGRUS_LEVEL" envDefault:"info"`
+	// LogrusFormat sets the logrus logging formatter
```

Comment:
let's also document the possible values here. thanks

Reply:
you bet